contracts/ref/cloud.yml

```diff
@@ -16895,6 +17002,27 @@ tags:
     - [InfluxDB stacks](https://docs.influxdata.com/influxdb/cloud/influxdb-templates/stacks/)
     - [InfluxDB templates](https://docs.influxdata.com/influxdb/cloud/influxdb-templates/)
   name: Templates
+- description: |
+    Retrieve specific users.
+
+    InfluxDB Cloud lets you invite and collaborate with multiple users in your organization.
+    To invite and remove users from your organization, use the InfluxDB Cloud user interface (UI);
+    you can't use the InfluxDB API to manage users in InfluxDB Cloud.
+    Once a user is added to your organization, you can use the
+    `GET /api/v2/users` and `GET /api/v2/users/USER_ID` API endpoints to
+    view specific members.
+
+    #### User sessions with authorizations
+
+    Optionally, you can scope an authorization (and its API token) to a user.
```

Comment:
is the authorization and API token always 1 to 1 relationship?

Reply:
> is the authorization and API token always 1 to 1 relationship?

Yes